src/utils.h

```diff
@@ -266,6 +266,15 @@ inline std::string &trim(std::string &s)
   return ltrim(rtrim(s));
 }
 
+inline bool ends_with(const std::string &s, const std::string &suffix)
```

Comment:
Sadly, I don't think we're on C++ 20 yet otherwise we could use ends_with:
https://en.cppreference.com/w/cpp/string/basic_string/ends_with

Reply:
yeah, I tried std::string::ends_with but it fails compilation. So, had to do this helper.